client/my-sites/plan-features/item.jsx

```diff
@@ -27,15 +28,25 @@ export default function PlanFeaturesItem( {
 		onMouseLeave( event.currentTarget, description );
 	};
 
+	const mouseEvents = {
+		onMouseEnter: handleOnMouseEvent,
+		onMouseLeave: handleOnMouseLeave
```

Comment:
Yes, totally. Also I believe it's outside of the scope of this change (I dont want another rabbithole of refactoring popover)